src/app/data/services/ajax_service.spec.ts

```diff
@@ -10,41 +10,72 @@ describe('AjaxService', function () {
 
   const service = new AjaxService();
   const newContentBlock = AjaxServiceMockData.newContentBlock();
-  const existingContentBlock = AjaxServiceMockData.existingContentBlock();
+  const existingContentBlock1 = AjaxServiceMockData.existingContentBlock1();
+  const existingContentBlock2 = AjaxServiceMockData.existingContentBlock2();
 
-  describe('#fetchContentBlock', function () {
-    it('fetches a content block with the right request method', function () {
+  describe('#fetchContentBlocks', function () {
+    it('fetches all content blocks with the right request method', function () {
+      spyOn(service, 'currentTime').and.returnValue(742.244);
       spyOn(jQuery, 'ajax').and.callFake(function (request:any) {
-        expect(request.url).toBe('http://renuo-cms-client.dev/v1/api-keyx/content_blocks/my-path');
         expect(request.type).toBe('get');
         expect(request.dataType).toBe('json');
-        return ajax_response(newContentBlock);
+        return ajax_response([existingContentBlock1, existingContentBlock2]);
       });
-      service.fetchContentBlock(new ContentBlock('', 'my-path', 'api-keyx', 'http://renuo-cms-client.dev')).then(() => {
+      service.fetchContentBlocks('api-keyx', 'http://renuo-cms-client.dev', true).then(() => null);
```

Comment:
`api-keyx` might look like a typo. you can rename it to `test-api-key`, but i can also live with it if you leave it